internal/config/config.go

```diff
@@ -16,6 +16,7 @@ const (
 	LabelKey               = "infra.weave.works/branch-planner"
 	LabelValue             = "true"
 	LabelPRIDKey           = "infra.weave.works/pr-id"
+	LabelOriginalNameKey   = "infra.weave.works/original-name"
```

Comment:
You mean the original object name?

Reply:
Don't we have owner reference set? I remember @squaremo and @opudrovs talked about it.

Reply:
Checked, no we don't.

Reply:
`source-terraform` may sounds a bit more descriptive, but I'm fine with `original-name` too.

Reply:
Based on this doc: https://sdk.operatorframework.io/docs/building-operators/ansible/reference/retroactively-owned-resources/#for-objects-which-are-not-in-the-same-namespace-as-the-owner-cr

using a similar naming for label seems logical: `infra.weave.works/primary-resource`
it can be even reused if we need for other types later.

Reply:
> we wouldn't be able to filter out directly in the list call, we would need to list all objects before manually filtering them

Is this using a shared cache (or could it reasonably be)? You can index the objects by owner and use the index in the query.

Reply:
Make it work, _then_ make it right :-)